godtools/App/Flows/Lesson/LessonFlow.swift

```diff
@@ -43,18 +43,31 @@ class LessonFlow: ToolNavigationFlow, Flow {
             appLanguage: appLanguage,
             toolTranslations: toolTranslations
         )
-              
+        
+        let initialPageOrPreviousProgress: MobileContentPagesPage?
```

Comment:
Something I didn't think about is in what cases would we not want to open a lesson to last progress? Was that something that was discussed?

Reply:
Disregard this.  I see it's falling back to last progress only when initialPage is null on initialization. I read it incorrectly when asking that question.